feat(Tile): Allow caption to be a React element (#1510)
react-native-training/react-native-elements: docs/tile.md

|  Type  | Default |
| :----: | :-----: |
| string **OR** React element or component |  none   |

---

### `captionStyle`

> Styling for the caption (optional); use this along with `caption` type `string`

Reviewer: Let's change the wording here to be bit more clear.

```suggestion
Styling for the caption (optional);  You only use this if `caption` is a string
```
